packages/start-slicemachine/package.json

```diff
@@ -26,7 +26,7 @@
 	],
 	"scripts": {
 		"build": "vite build",
-		"dev": "vite build --watch --mode development",
```

Comment:
From my understanding, we need to keep this (in both packages) because we're not using `vite dev` to develop but `vite build` with a watch flag, which uses `production` mode as a default: https://vitejs.dev/guide/env-and-mode.html#modes